src/pages/billing/NewBillingProjectWizard/StepWizard/StepFields.ts

```diff
@@ -0,0 +1,58 @@
+import { CSSProperties, ReactNode } from 'react'
+import { div, fieldset, form, h, legend } from 'react-hyperscript-helpers'
+import { FormLabel } from 'src/libs/forms'
+
+
+interface StepFieldsProps {
+  children?: ReactNode[]
+  style?: CSSProperties
+}
+
+export const StepFields = ({ children = [], style, disabled = false }: StepFieldsProps & { disabled?: boolean }) => fieldset({
+  disabled,
+  style: {
+    border: 'none',
+    margin: 0,
+    padding: 0,
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignContent: 'center',
+    width: '100%',
+    ...style
+  }
+}, [children])
+
+export const StepFieldLegend = ({ children = [], style }: StepFieldsProps) => legend({
+  style: {
+    fontSize: 14,
+    lineHeight: '22px',
+    whiteSpace: 'pre-wrap',
+    marginTop: '0.25rem',
+    marginRight: '1rem',
+    float: 'left',
+    ...style
+  }
+}, children)
+
+export const StepFieldForm = ({ children = [], style }: StepFieldsProps) => form({
+  style: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignContent: 'center',
+    justifyContent: 'space-between',
+    width: '100%',
+    ...style
+  }
+}, children)
+
+interface LabeledFieldProps extends StepFieldsProps {
+  formId: string
+  label: ReactNode
+  required?: boolean
+}
+
+export const LabeledField = ({ label, formId, required = false, children = [], style }: LabeledFieldProps) => div({ style: { display: 'flex', flexDirection: 'column', ...style } }, [
+  h(FormLabel, { htmlFor: formId, required }, [label]),
+  children
```

Comment:
Note that if you are able to put the children within the label, you don't need to link them together with an ID. So for example, the following is perfectly OK and avoids the need to create an ID:

````
<label>Address:<input></input></label>

Reply:
I think I had to do this because the project-wide `ValidatedInput` component uses some absolute positioning (and maybe some other witchery, too), so this was the only way I could get it to work.

The `CreateGCPBillingProject` component does essentially the same thing, I'm assuming for the same reason.

Reply:
OK, that's fine. Just wanted to let you know because the general pattern in terra-ui is to always use an _id_, but that's not necessary if you can nest the components.